### ISSUE TITLE
Validation set not used?

### ISSUE THREAD

**Issue by yiwang454** [DushyantChauhan/NAACL-19-CIM#1]
Hi, @DushyantChauhan 

I just started to work on emotion recognition using CMU MOSEI. Your work on this dataset seems to have a quite scientific evaluation of your model using this dataset according to your paper, and thank you for putting the code here. I tried to go through your code and I'm wondering if you only used train and test sets of the MOSEI dataset (according to the code below)? Pls correct me if I misunderstood about this. If you've included validation set to choose the code but not shows it in the code, how did you use it ? (like mixing it with train and held a part of the data out each time or use the given validation set all the time?)

'''
        history = model.fit([train_text, train_audio, train_video], [senti_train_label, emo_train_label],
                            epochs=50,
                            batch_size=16,
                            sample_weight=[train_mask, train_mask],
                            shuffle=True,
                            callbacks=[metrics_callback, bestModel_sentiment, bestModel_emotion],
                            validation_data=([test_text, test_audio, test_video], [senti_test_label,emo_test_label], [test_mask,test_mask]),
                            verbose=1)
'''
Also, in your paper you said 'We optimize and cross-validate both the evaluation metrics', and I was a bit confused about the cross validation part. Does that mean you varied training and test sets when evaluating those metrics? If yes, then how did you assign training and test sets?

Thank you for your response in advance.
Best regards,
Yi

**Comment by GussailRaat** [comment]
Hello,

That use of test set is just to check the behaviour of learning. Use valid set instead.
